database/src/seeds/03_basic_project_survey_setup.ts

```diff
@@ -44,6 +46,11 @@ export async function seed(knex: Knex): Promise<void> {
     `);
   }
 
+  // Insert access requests
+  for (let i = 0; i < 15; i++) {
+    await knex.raw(`${insertAccessRequest()}`);
```

Comment:
mega nit: Promise.all to improve seed execution time

Reply:
Fixed